src/components/manage/Form/Form.jsx

```diff
@@ -82,6 +82,9 @@ class Form extends Component {
     description: PropTypes.string,
     visual: PropTypes.bool,
     blocks: PropTypes.arrayOf(PropTypes.object),
+    isFormSelected: PropTypes.bool,
+    onSelectForm: PropTypes.func,
```

Comment:
This has a smell. What's the idea?

Reply:
isFormSelected is a boolean that tells if form is selected. This is needed when we are in babel view and we focus on a field of source translation object to view, for example, the configuration of a block in sidebar. 

onSelectForm is a function to set the state variable 'formSelected' to know in Edit or Add mode, which form is selected, again for the reason of showing the correct data in the sidebar.

You could better understand if you look at Edit.jsx and Add.jsx components